packages/mobile/src/screens/profile-screen/ProfileHeaderV2/SupportingList.tsx

```diff
@@ -81,6 +81,7 @@ export const SupportingList = () => {
       <SupportingTile
         style={styles.singleSupporterTile}
         supporting={supportingSorted[0]}
+        onPressScaleTo={0.985}
```

Comment:
even if the the prop `scaleTo` is not clear for tile, i think it's best to use the same prop names for things that are the same type of component. so since SupportingTile is a tile, then `scaleTo` would be preferred by me to keep api consistent. You can also update SupportingTile props in a potentially cleaner way by doing

```
type SupportingTileProps = TileProps & {...}
```